dali/operators/reader/loader/coco_loader.cc

```diff
@@ -97,20 +115,42 @@ void SaveFilenamesToFile(const ImageIdPairs &image_id_pairs, const std::string p
 template <typename T>
 void LoadFromFile(std::vector<T> &output, const std::string path) {
   std::ifstream file(path);
-  DALI_ENFORCE(file.good(),
-               make_string("CocoReader failed to read preprocessed annotation data from ", path));
+  output.clear();
+  if (!file.good())
+    return;
 
   unsigned size;
   file.read(reinterpret_cast<char*>(&size), sizeof(unsigned));
   output.resize(size);
   file.read(reinterpret_cast<char*>(output.data()), size * sizeof(T));
 }
 
+template <>
+void LoadFromFile(std::vector<RLEMask> &output, const std::string path) {
+  std::ifstream file(path);
+  output.clear();
+  if (!file.good())
+    return;
+
+  unsigned size;
+  file.read(reinterpret_cast<char*>(&size), sizeof(unsigned));
```

Comment:
If we are already touching this can you:
- check `size` is smaller than the file size and >=0
- `m * sizeof(uint)` is smaller than the file size and >= 0
- `h` and `w` are sane as well?

Do the same for other `LoadFromFile` variants?